### PR TITLE
[WIP] Migration to Scala 2.13.

### PR DESCRIPTION
**Pull request by rpiaggio**
Everything's working, but do not merge yet.

We need to wait for release of `jQuery` facade.

### CODE REVIEW

File: modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ import mouse.all._
 import seqexec.engine._
 import seqexec.engine.Action.ActionState
 import seqexec.model.enum.{Instrument, Resource}
-import seqexec.model._
+import seqexec.model.{Progress => _, _}
```
Comment (rpiaggio) [inline modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala]: Scala 2.13 now favors imported types over ones in the current package.

File: project/Common.scala [hunk 2]
```diff
@@ -9,8 +9,7 @@ import com.timushev.sbt.updates.UpdatesPlugin.autoImport._
 object Common {
   lazy val commonSettings = Seq(
     // Workaround for https://github.com/sbt/sbt/issues/4109
-    initialCommands += "jline.TerminalFactory.get.init\n",
-    scalaVersion := Settings.LibraryVersions.scalaVersion,
```
Comment (rpiaggio) [inline project/Common.scala]: Now set globally in `inThisBuild` in `build.sbt`.

**Comment by cquiroz** [comment]
pureconfig looks available for 2.13, what’s missing?

**Comment by rpiaggio** [comment]
> pureconfig looks available for 2.13, what’s missing?

The latest release `v0.12.2` excludes Scala 2.13 in the `http4s` module, which we depend on: https://github.com/pureconfig/pureconfig/blob/v0.12.2/modules/http4s/build.sbt (this allows reading `http4s`'s `Uri`s from the config)

It's fixed in their `master` but they haven't released yet.

**Comment by tpolecat** [comment]
@rpiaggio I'll poke them and see if they can do a release.

**Comment by tpolecat** [comment]
Ah you already did.

**Comment by rpiaggio** [comment]
> Ah you already did.

A request from you might pull more weight ;)

**Pull request merged**